NMV_strongs_Genesis.json

```diff
@@ -0,0 +1 @@
H559"]], [["«اینک"], ["من", null], ["عهد", "H1285"], ["خویش"], ["را"], ["با"], ["شما"], ["و"], ["پس", "H310"], ["از"], ["شما"], ["با"], ["فرزندان"], ["شما"], ["استوار"], ["می‌سازم،", "H6965"]], [["و"], ["نیز"], ["با"], ["هر", "H3605"], ["جانداری", "H2416"], ["که"], ["با"], ["شما"], ["باشد،"], ["از"], ["پرندگان", "H5775"], ["و"], ["چارپایان"], ["و"], ["همۀ"], ["جانوران", "H2416"], ["زمین،", "H776"], ["یعنی"], ["با"], ["همۀ"], ["آنها"], ["که"], ["از"], ["کشتی", "H8392"], ["بیرون"], ["آمدند؛"], ["این"], ["برای"], ["همۀ"], ["جانوران"], ["زمین", "H776"], ["خواهد"], ["بود.", "H2416"]], [["من", null], ["عهد", "H1285"], ["خود"], ["را"], ["با"], ["شما"], ["استوار"], ["می‌گردانم"], ["که"], ["دیگر", "H5750"], ["هرگز", "H3808"], ["هر"], ["ذی‌جسد"], ["به"], ["آب", "H4325"], ["توفان"], ["هلاک"], ["نشود،", null], ["و"], ["دیگر"], ["هرگز"], ["توفانی"], ["نباشد", null], ["که"], ["همۀ"], ["زمین", "H776"], ["را"], ["ویران"], ["کند.»"]], [["و"], ["خدا", "H430"], ["گفت:", "H559"], ["«این"], ["است", "H589"], ["نشان", "H5414"], ["عهدی", "H1285"], ["که"], ["من", "H589"], ["میان", "H966"], ["خود"], ["و"], ["شما"], ["و"], ["هر", "H3605"], ["جانداری"], ["که"], ["با"], ["شماست،"], ["برای"], ["همۀ"], ["نسلهای", "H1755"], ["آینده", "H5769"], ["می‌بندم."]], [["رنگین‌کمان"], ["خود"], ["را"], ["در"], ["ابر", "H6051"], ["قرار"], ["داده‌ام،"], ["و"], ["آن"], ["نشان", "H5414"], ["عهدی", "H1285"], ["خواهد"], ["بود"], ["که"], ["میان", "H966"], ["من"], ["و"], ["زمین", "H776"], ["است."]], [["هرگاه"], ["ابرها"], ["را"], ["بر", "H6049"], ["فراز"], ["زمین", "H776"], ["بگسترانم"], ["و"], ["رنگین‌کمان"], ["در"], ["ابرها", "H6051"], ["پدیدار"], ["شود،"]], [["آنگاه"], ["عهد", "H1285"], ["خود"], ["را"], ["با"], ["شما"], ["و"], ["هر"], ["جاندار", null], ["ذی‌جسد"], ["به"], ["یاد"], ["خواهم"], ["آورد."], ["و"], ["آبها", "H4325"], ["دیگر", "H5750"], ["هرگز", "H3808"], ["سیل", "H3999"], ["نخواهد"], ["شد", null], ["تا"], ["هر", "H3605"], ["ذی‌جسد"], ["را"], ["هلاک"], ["کند."]], [["هرگاه"], ["رنگین‌کمان"], ["در"], ["ابرها", "H6051"], ["پدیدار"], ["شود،"], ["آن"], ["را"], ["خواهم"], ["دید", "H7200"], ["و"], ["عهد", "H1285"], ["جاودانی"], ["میان", "H996"], ["خدا", "H430"], ["و"], ["هر", "H3605"], ["جاندار", "H2416"], ["ذی‌جسد"], ["را"], ["که"], ["بر", "H5921"], ["زمین", "H776"], ["است"], ["به", "H5921"], ["یاد"], ["خواهم"], ["آورد.»"]], [["پس"], ["خدا", "H430"], ["به", "H5921"], ["نوح"], ["فرمود:", "H559"], ["«این"], ["است"], ["نشان", "H6965"], ["عهدی", "H1285"], ["که"], ["در"], ["میان", "H966"], ["خود"], ["و"], ["هر"], ["ذی‌جسدی"], ["که"], ["بر", "H776"], ["زمین", "H776"], ["است،"], ["استوار"], ["ساخته‌ام.»", "H6965"]], [["پسران", "H1121"], ["نوح"], ["که", null], ["از"], ["کشتی", "H8392"], ["بیرون"], ["آمدند،"], ["سام"], ["و"], ["حام"], ["و"], ["یافِث"], ["بودند.", null], ["حام"], ["پدر", "H1"], ["کنعان", "H3667"], ["بود.", null]], [["اینان"], ["سه", "H7969"], ["پسر", "H1121"], ["نوح"], ["بودند", null], ["و"], ["مردمان"], ["تمامی"], ["جهان", "H776"], ["از"], ["ایشان"], ["منشعب"], ["شدند.", null]]]]}}
```

Comment:
It would be good to have nothing at all output, rather than `null` as it would save space :)